### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Rebuild projects page as per [the new design](https://warthogs.atlassian.net/browse/WD-3720)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Compare the page to the design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3720

## Screenshots

image

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@clem any comments on this from dev perspective?

**Comment by lyubomir-popov** [comment]
All done, please have another look @juanruitina. Any dev/QA comments @ClementChaumel ?

**Comment by juanruitina** [comment]
LGTM!

**Comment by ClementChaumel** [comment]
I removed Bazaar to keep up with https://github.com/canonical/canonical.com/pull/921

**Review by ClementChaumel — APPROVED** [review]
Awesome 🤙

File: templates/projects/index.html [hunk 1]
```diff
@@ -5,170 +5,134 @@
...
 
-<section class="p-strip--light is-deep">
+<section class="p-section">
   <div class="row">
-    <div class="col-12">
-      <h2 class="p-muted-heading">Some projects we care about</h2>
+    <h2 class="p-muted-heading">Some projects we care about</h2>
+    <hr class="p-rule--highlight">
+  </div>
+  <div class="row p-section">
```
Comment (ClementChaumel) [inline templates/projects/index.html:28]: Nested sections here too FYI

**Pull request merged**